drivers/feetech/include/feetech.h

```diff
@@ -0,0 +1,101 @@
+#ifndef FEETECH_H
+#define FEETECH_H
+
+#include "feetech_protocol.h"
+#include "feetech_reader.h"
+#include "feetech_writer.h"
+
+#include "periph/uart.h"
+
+#define FEETECH_STREAM_TYPE uart_t
+#define FEETECH_STREAM_INIT(stream) uart_init(stream, 1000000, 0, 0) // TODO
+#define FEETECH_STREAM_WRITE(stream, buffer, size) uart_write(stream, buffer, size)
+#define FEETECH_STREAM_READ(stream, buffer, size) ((void)buffer, size) // TODO
+
+typedef FEETECH_STREAM_TYPE feetech_stream_t;
+typedef uint8_t feetech_device_t;
+
+void feetech_init(feetech_stream_t stream) {
+  FEETECH_STREAM_INIT(stream);
```

Comment:
OK.
I did not follow the coding conventions for the moment because I just translated from C++.
It will be done !